### ISSUE TITLE
Icon to Submit for General use of the Community - BULMA Brand Icon

### ISSUE THREAD

**Issue by Dinuz** [FortAwesome/Font-Awesome#13076]
fontawesome.findIconDefinition({prefix: 'fab', iconName: 'bulma'})

  
<svg width="46" height="67" viewBox="0 0 46 67" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="Canvas" fill="none">
<path id="Path" fill-rule="evenodd" clip-rule="evenodd" d="M 0 45.4337L 4.13034 16.5213L 20.6517 0L 41.3034 20.6517L 28.9124 33.0427L 45.4337 49.564L 20.6517 66.0854L 0 45.4337Z" transform="translate(0.00512695 0.174698)" fill="#00D1B2"/>
</g>
</svg>

Can we add to the whole package?

**Comment by jgthms** [comment]
Actually, the official "B" symbol is this one:

```xml
<svg width="220px" height="320px" viewBox="0 0 220 320" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <polygon points="0 220 20 80 100 0 200 100 140 160 220 240 100 320" fill="#00D1B2"></polygon>
</svg>
```

**Comment by Dinuz** [comment]
@jgthms I believe you are correct:) 
I fell in love with your framework recently. Yesterday I was looking for the vector version of the logo, couldn’t find it on the official site(just the made with in png:(( )...so I started looking around! Guess what? I got that svg from I believed was your blog.

The version you provided is more elegant and short, I am happy you directly provided it!

**Comment by jgthms** [comment]
Yeah I should create a "Logos" or "Press" page with the logo in different formats, and some screenshots too!

**Comment by My1** [comment]
I would also totally love a bulma icon in here, also @jgthms the logo in different formats thing is always a good idea.

**Comment by My1** [comment]
I guess I can make FA's work a bit easier.

the icons on the svg side are 512px high, so we could make it work for the icon:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 352 512">
  <path d="M0 352L32 128L160 0L320 100L224 256L352 384L160 512Z"></path>
</svg>
```
I grabbed the header of svgs/solid/square-full.svg and calculated the points for a 512h version, as well as made it into a path, as the square full was using that instead of a poly so I guess path is preferred here.
is that okay the way I done it @jgthms ?

**Comment by jgthms** [comment]
Just multiply every value by `1.6`:

```html
<svg width="352px" height="512px" viewBox="0 0 352 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <polygon points="0 352 32 128 160 0 320 160 224 256 352 384 160 512" fill="#00D1B2"></polygon>
</svg>
```

**Comment by My1** [comment]
this is exactly how I got the numbers, (except I missed one number, which I edited)

I just also made it into how the SVGs are on the Fontawesome side of things, (e.g. no fill needed)
so ideally they could literally just slap in the SVG into the repo (except for erasing some spaces and newlines), trying to make their work easier.

**Comment by jgthms** [comment]
Ah I see. Thanks for your help!

**Comment by My1** [comment]
no problem. I love bulma and chucked in your SVG into a custom micro-build of FA (7 icons incl Bulma) using icomoon just so I can throw on some credit for both Font Awesome and Bulma because they both are awesome.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
